resources/js/phonefield.js

```diff
@@ -28,7 +28,7 @@ function phoneMask(fields) {
     function addMask(iti, inputQuery){
         let selectedCountryData = iti.getSelectedCountryData();
         let newPlaceholder = intlTelInputUtils.getExampleNumber(selectedCountryData.iso2, true, intlTelInputUtils.numberFormat.INTERNATIONAL);
-        $(inputQuery).inputmask({ mask: newPlaceholder.replace(/[0-9+]/ig,'9'), keepStatic: false });
+        // $(inputQuery).inputmask({ mask: newPlaceholder.replace(/[0-9+]/ig,'9'), keepStatic: false });
```

Comment:
If you comment this out the function won't work.

Reply:
i removed that line

Reply:
i mean remove function block, because if you removed that line this function not work